src/main/java/uk/gov/hmcts/divorce/common/validation/ValidationUtil.java

```diff
@@ -132,6 +132,7 @@ public static boolean isPaymentIncomplete(CaseData caseData) {
     }
 
     public static boolean hasAwaitingDocuments(CaseData caseData) {
+        // TODO - use .equals() for string comparison instead of ==
```

Comment:
I can't remember whether we did but I think we were going to do a .toBool on YesOrNo that would be easier

Reply:
Used the .toBoolean() method in the YesOrNo model 👍